### PR TITLE
add: typography + nav + rating component

### PR DESCRIPTION
**Pull request by nayyyhaa**
Added navbar, typography & ratings component.

### CODE REVIEW

**Review by partha8 — APPROVED** [review]
LGTM!

**Review by VLeads — APPROVED** [review]
Code and UI looks good.

**Review by sainath7878 — APPROVED** [review]
Looks good to me. Neat and clean UI :)

**Comment by nayyyhaa** [comment]
Thanks @partha8 , @VLeads & @sainath7878 for reviewing. Merging the changes.

File: styles/components-styles/badge.css [hunk 1]
```diff
@@ -45,20 +45,20 @@
   height: 3.5rem;
```
Comment (govindappaarun) [inline styles/components-styles/badge.css:45]: setting height is not a best practice, and you are unsetting in the other forms of txt-badge, 
any reason you need to set here a fixed height?

**Pull request merged**